htop.c

```diff
@@ -48,6 +48,7 @@ static void printHelpFlag(void) {
          "-u --user[=USERNAME]        Show only processes for a given user (or $USER)\n"
          "-U --no-unicode             Do not use unicode but plain ASCII\n"
          "-V --version                Print version info\n"
+         "-H --highlight-new-old      Highlight new and old processes\n"
```

Comment:
Not quite happy with `--highlight-new-old` yet. What about something like `--highlight-diff`?

Reply:
Also this could take the optional delay to keep things highlighted, default as configured in `.htoprc` if not given.

Reply:
> Not quite happy with `--highlight-new-old` yet. What about something like `--highlight-diff`?

or `--highlight-changes`

Reply:
Changed to `--highlight-changes`